hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/hudi/functional/TestRecordLevelIndexWithSQL.scala

```diff
@@ -44,6 +46,28 @@ import scala.util.Using
 class TestRecordLevelIndexWithSQL extends RecordLevelIndexTestBase {
   val sqlTempTable = "tbl"
 
+  @ParameterizedTest
+  @ValueSource(booleans = Array(true, false))
+  def testRLICreationUsingSQL(isPartitioned: Boolean): Unit = {
```

Comment:
why do we need a new test. why can't we add validations to one of the existing test only?
`TestPartitionedRecordLevelIndex`

Reply:
This type of test does not exist in RLI test suite. We are directly calling the index creation APIs here.